shell/platform/darwin/ios/framework/Source/FlutterEmbedderKeyResponder.mm

```diff
@@ -356,15 +354,15 @@ @interface FlutterEmbedderKeyResponder ()
  *
  * Set by the initializer.
  */
-@property(nonatomic, copy, readonly) FlutterSendKeyEvent sendEvent;
+@property(nonatomic, readonly) FlutterSendKeyEvent sendEvent;
```

Comment:
nit: iirc Apple still recommend using `copy` for blocks in ARC, despite that `retain/strong` behaves the same. It's more of a hint to the reader.

Reply:
From https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/WorkingwithBlocks/WorkingwithBlocks.html

> Note: You should specify copy as the property attribute, because a block needs to be copied to keep track of its captured state outside of the original scope. This isn’t something you need to worry about when using Automatic Reference Counting, as it will happen automatically, but it’s best practice for the property attribute to show the resultant behavior. For more information, see [Blocks Programming Topics](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Blocks/Articles/00_Introduction.html#//apple_ref/doc/uid/TP40007502).

Reply:
Ah, didn't know about that "it’s best practice for the property attribute to show the resultant behavior" recommendation.  I'll change it back I guess... 

https://clang.llvm.org/docs/AutomaticReferenceCounting.html#blocks